pkg/profileparser/profileparser.go

```diff
@@ -541,6 +541,23 @@ func ParseRulesAndDo(contentDom *xmlquery.Node, stdParser *referenceParser, pb *
 	questionsTable := utils.NewOcilQuestionTable(contentDom)
 	defTable := utils.NewDefHashTable(contentDom)
 
+	allValues := xmlquery.Find(contentDom, "//xccdf-1.2:Value")
+	valuesList := make(map[string]string)
+	for _, variable := range allValues {
+		for _, val := range variable.SelectElements("//xccdf-1.2:value") {
+			if val.SelectAttr("hidden") == "true" {
+				// this is typically used for functions
+				continue
+			}
+			if val.SelectAttr("selector") == "" {
```

Comment:
Can you remind me what the empty selector means?

Reply:
Not present empty selector means the value is not one of selection values, and it is a default value